docs/reference/aggregations.asciidoc

```diff
@@ -3,117 +3,433 @@
 
 [partintro]
 --
-The aggregations framework helps provide aggregated data based on a search query. It is based on simple building blocks
-called aggregations, that can be composed in order to build complex summaries of the data.
+An aggregation summarizes your data as metrics, statistics, or other analytics.
+Aggregations help you answer questions like:
 
-An aggregation can be seen as a _unit-of-work_ that builds analytic information over a set of documents. The context of
-the execution defines what this document set is (e.g. a top-level aggregation executes within the context of the executed
-query/filters of the search request).
+* What's the average load time for my website?
+* Who are my most valuable customers based on transaction volume?
+* What would be considered a large file on my network?
+* How many products are in each product category?
 
-There are many different types of aggregations, each with its own purpose and output. To better understand these types,
-it is often easier to break them into four main families:
+{es} organizes aggregations into three categories:
 
-<<search-aggregations-bucket, _Bucketing_>>::
-				A family of aggregations that build buckets, where each bucket is associated with a _key_ and a document
-				criterion. When the aggregation is executed, all the buckets criteria are evaluated on every document in
-				the context and when a criterion matches, the document is considered to "fall in" the relevant bucket.
-				By the end of the aggregation process, we'll end up with a list of buckets - each one with a set of
-				documents that "belong" to it.
+* <<search-aggregations-metrics,Metric>> aggregations that calculate metrics,
+such as a sum or average, from field values.
 
-<<search-aggregations-metrics, _Metric_>>::
-				Aggregations that keep track and compute metrics over a set of documents.
+* <<search-aggregations-bucket,Bucket>> aggregations that
+group documents into buckets, also called bins, based on field values, ranges,
+or other criteria.
 
-<<search-aggregations-matrix, _Matrix_>>::
-                A family of aggregations that operate on multiple fields and produce a matrix result based on the
-                values extracted from the requested document fields. Unlike metric and bucket aggregations, this
-                aggregation family does not yet support scripting.
+* <<search-aggregations-pipeline,Pipeline>> aggregations that take input from
+other aggregations instead of documents or fields.
 
-<<search-aggregations-pipeline, _Pipeline_>>::
-				Aggregations that aggregate the output of other aggregations and their associated metrics
+[discrete]
+[[run-an-agg]]
+=== Run an aggregation
+
+{es} runs aggregations as part of a <<search-your-data,search>>. To run an
+aggregation, use the <<search-search,search API>>'s `aggs` parameter. The
+following search runs `my-agg-name`, a
+<<search-aggregations-bucket-terms-aggregation,terms aggregation>> on
+`my-field`:
```

Comment:
The agg name is pretty self-explanatory--I don't think it needs to be called out.
```suggestion
following search runs  a
<<search-aggregations-bucket-terms-aggregation,terms aggregation>> on
`my-field`:
```